Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:7
+
+# Establishing Working Directory
+WORKDIR /usr/src/app
```

Comment:
we could use an app or src directly because mounting a folder structure following /usr will be confusing